Makefile

```diff
@@ -9,7 +9,15 @@ pg_squeeze--1.4--1.5.sql pg_squeeze--1.5--1.6.sql
 DOCS = pg_squeeze.md
 
 REGRESS = squeeze
+REGRESS_OPTS = --temp-config=postgresql.conf
 
+ifdef USE_PGXS
```

Comment:
All of our extensions default to `PGXS` due to packaging reasons. Your contribution changes that, which is not good.